cipher/cipher_engine.py

```diff
@@ -0,0 +1,86 @@
+from sans_program_project.cipher.cipher import Cipher
+
+
+def _reverseCipher(plaintext):
+    # ciphertext = list(reversed(plaintext))
+    # return ''.join(ciphertext)
+    ciphertext = plaintext[::-1]
+    return ciphertext
+
+
+def _caesar_encrypt(my_bytes, n_shift):
```

Comment:
why outside the class, do you use it anywhere else?

Reply:
No, it won't be used in other places. The reason I did this is because it makes the class better organized. Logic is more clear to me. "_caesar_encrypt" is a helper function. I borrowed this pattern from some packages in sklearn, but it's certainly up to discussion.